buckets.yaml

```diff
@@ -1,17 +1,18 @@
-# A mapping of bucket URLs to bucket metadata
+# A mapping of bucket URLs to bucket metadata.
+# Please include a contact for new prefixes!
```

Comment:
Should we prescribe what kind of contact information, e.g., Slack user ID, email?